src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,83 @@
-export const Pagination = () => {};
+interface Props {
+  currentPage: number;
+  itemsPerPage: number;
+  totalItems: number;
+  paginate: (pageNumber: number) => void;
+}
+
+export const Pagination: React.FC<Props> = ({
+  currentPage,
+  itemsPerPage,
+  totalItems,
+  paginate,
+}) => {
+  const pageNumbers = [];
+
+  for (let i = 1; i <= Math.ceil(totalItems / itemsPerPage); i += 1) {
+    pageNumbers.push(i);
+  }
+
+  const handleClick = (pageNumber: number) => {
+    paginate(pageNumber);
+  };
+
+  const handleNextPage = () => {
+    if (currentPage < pageNumbers.length) {
+      paginate(currentPage + 1);
+    }
+  };
+
+  const handlePrevPage = () => {
+    if (currentPage > 1) {
+      paginate(currentPage - 1);
+    }
+  };
+
+  return (
+    <nav>
+      <ul className="pagination">
+        <li className={`page-item ${currentPage === 1 ? 'disabled' : ''}`}>
```

Comment:
- You compare `currentPage === 1` two times, create a variable
- Use classnames library